tests/test_cli.py

```diff
@@ -387,21 +393,69 @@ def test_zenodo_valid_but_not_open_access(script_runner):
     assert "This record does not have Open Access files. Verify the Access rights of the record" in ret.stderr
 
 
-def test_export(script_runner):
+def test_export_relative_path(script_runner):
     with tempfile.TemporaryDirectory() as tmp:
         gpkg_file = os.path.join(tmp, "export_file.gpkg")
```

Comment:
This seems to be the same as in `test_export_absolute_path`. Please check.

Maybe you can run the `script_runner` within the temp directory to have a relative path that you don't have to clean up?

Reply:
Fixed ! Thanks !